reference_files/generate/generate.py

```diff
@@ -155,8 +155,8 @@ def generate(version):
         filename = os.path.join(outdir, name)
         func(filename + ".asdf")
         with asdf.open(filename + ".asdf") as af:
-            af.resolve_and_inline()
-            af.write_to(filename + ".yaml")
+            af.resolve_references()
```

Comment:
This replaces the only use of `resolve_and_inline` I'm able to find in any package outside of asdf. See https://github.com/asdf-format/asdf/pull/1690 for a description of the `resolve_and_inline` deprecation which includes the suggestion to update code similar to the update in this PR (replace `resolve_and_inline` with `resolve_references` and `all_array_storage='inline'`).